cmd/minikube/cmd/stop.go

```diff
@@ -42,28 +47,54 @@ itself, leaving all files intact. The cluster can be started again with the "sta
 	Run: runStop,
 }
 
-// runStop handles the executes the flow of "minikube stop"
-func runStop(cmd *cobra.Command, args []string) {
-	cname := ClusterFlagValue()
+func init() {
 
-	api, cc := mustload.Partial(cname)
-	defer api.Close()
+	stopCmd.Flags().BoolVar(&stopAll, "all", false, "Set flag to stop all profiles")
```

Comment:
nit: I would change this to
"stop all profiles (clusters)"

Reply:
Updated!